### PR TITLE
Only fire Multiselect input function once

### PR DESCRIPTION
**Pull request by raimund-schluessler**
This PR aims to fix the double call of the input callback for the Multiselect. I'd like to add a jest test for this as well.

Since jest is not configured yet for this repo, I wonder whether I should fix the Multiselect bug in a different issue and only set up jest here?

Locally, jest runs fine already, but it's not run by the Github actions yet.

### CODE REVIEW

**Comment by raimund-schluessler** [comment]
I now added a test which reproduces the issue that the input function is called twice. Will proceed to fix the actual issue now.

**Comment by raimund-schluessler** [comment]
@nextcloud/vuejs @skjnldsv I set up the jest test environment to be run with GitHub actions, added a test to check for the bug (multiselect input called twice) and fixed the bug. The added jest test would fail before this PR, but passes now. I also tested if the change fixes https://github.com/nextcloud/tasks/issues/778 and it works fine there.

Would be great to get a review, so we can get the fix in.

File: .github/workflows/npm-test.yml [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+    strategy:
+      matrix:
+        node-version: [11.x]
+    steps:
+      - uses: actions/checkout@v1
+      - name: Use Node.js ${{ matrix.node-version }}
+        uses: actions/setup-node@v1
+        with:
+          node-version: ${{ matrix.node-version }}
+      - name: npm install
```
Comment (skjnldsv) [inline .github/workflows/npm-test.yml]: This is stupid, but can I ask if we can have the same case for the tests names? :see_no_evil: 
image

```suggestion
      - name: Install dependencies
```

File: .github/workflows/npm-test.yml [hunk 2]
```diff
@@ -0,0 +1,26 @@
+name: Test
+
+on:
+  pull_request:
+    branches:
+      - master
+      - stable*
+
+jobs:
+  build:
```
Comment (skjnldsv) [inline .github/workflows/npm-test.yml]:
```suggestion
  jest:
```

File: .github/workflows/npm-test.yml [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        node-version: [11.x]
+    steps:
+      - uses: actions/checkout@v1
+      - name: Use Node.js ${{ matrix.node-version }}
```
Comment (skjnldsv) [inline .github/workflows/npm-test.yml]:
```suggestion
      - name: Setup node ${{ matrix.node-version }}
```

File: .github/workflows/npm-test.yml (hunk 1 above)
Comment (raimund-schluessler) [reply .github/workflows/npm-test.yml]: Fine with me, I also wondered how I should name the tests 😃

Comment (skjnldsv) [reply .github/workflows/npm-test.yml]: I standardised it for server/contacts & photos so far :woman_shrugging: 
https://github.com/nextcloud/photos/tree/master/.github/workflows

Comment (raimund-schluessler) [reply .github/workflows/npm-test.yml]: Then I guess we need another PR for nextcloud-vue 😉

**Pull request merged**